tap_github/schemas/workflow_runs.json

```diff
@@ -0,0 +1,159 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "_sdc_repository": {
+      "type": [
+        "string"
+      ]
+    },
+    "id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "node_id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "head_branch": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "head_sha": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "run_number": {
```

Comment:
This should be an integer based on the documentation

Reply:
you're right! Done!